src/components/search/search-result.js

```diff
@@ -3,12 +3,29 @@ import { Link } from 'gatsby'
 import DetailText from '~components/common/detail-text'
 import resultStyle from './search-result.module.scss'
 
-const SearchResult = ({ title, url, children = false }) => (
+const SearchResult = ({
+  title,
+  url,
+  publishDate,
+  author,
+  children = false,
+}) => (
   <div className={resultStyle.result}>
     <h4>
       <Link to={url}>{title}</Link>
     </h4>
-    {children && <DetailText>{children}</DetailText>}
+    <div>{children}</div>
+    {(publishDate || author) && (
+      <DetailText>
+        {publishDate && (
+          <span className={resultStyle.publishDate}>{publishDate}</span>
+        )}
+        {publishDate && author && (
+          <span dangerouslySetInnerHTML={{ __html: ' &middot; ' }} />
```

Comment:
Maybe you don't need to use `dangerouslySetInnerHTML` here, normally if you copy/paste the dot from somewhere it's rendered, you'll paste it without having to mess with HTML.
`dangerouslySetInnerHTML` is great for marked content (or content coming from a WYSIWYG in other projects)

Reply:
For sure, that makes sense, good point. I struggled with getting the HTML entity to render, and settled upon this solution. Would definitely rather avoid this.